app/presenters/updatable.rb

```diff
@@ -20,13 +25,21 @@ def history
     end
   end
 
-private
+  # The direction of change history isn’t guaranteed
+  # https://github.com/alphagov/govuk-content-schemas/issues/545
+  def reverse_chronological_change_history
+    change_history.sort_by { |item| DateTime.parse(item[:timestamp]) }.reverse
+  end
 
   def any_updates?
-    if (first_public_at = content_item["details"]["first_public_at"]).present?
+    if first_public_at.present?
```

Comment:
It can be blank for first drafts (I don't know if government-frontend is used to preview content in the draft stack though).